Editors/ImportExportEditor/Editors.ImportExport/Exporting/Exporters/RmvToGltf/Helpers/GltfSkeletonBuilder.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System.Numerics;
+using Editors.ImportExport.Common;
+using Editors.ImportExport.Exporting.Exporters.GltfSkeleton;
+using Editors.Shared.Core.Services;
+using GameWorld.Core.Animation;
+using GameWorld.Core.SceneNodes;
+using Shared.Core.PackFiles;
+using Shared.GameFormats.Animation;
+using SharpGLTF.Schema2;
+using SysNum = System.Numerics;
+
+namespace Editors.ImportExport.Exporting.Exporters.RmvToGltf.Helpers
+{
+    public class ProcessedGltfSkeleton
+    {
+        public required List<(Node, Matrix4x4)> Data { get; set; }
+    }
+
+    public class GltfSkeletonBuilder
```

Comment:
I dont think its a good idea to split animation and skeleton logic into two files. 
The code seems very related and if one failes the other would also fail. 
If you make changes to one, you almost always will need to make changes to the other. They are highly coupled and should therefor probably be merged